src/styled/json.ts

```diff
@@ -1,11 +1,12 @@
 // tslint:disable restrict-plus-operands
 
 import chalk from 'chalk'
+import util from 'util'
 
 import cli from '..'
 
 export default function styledJSON(obj: any) {
-  const json = JSON.stringify(obj, null, 2)
+  const json = util.inspect(obj)
```

Comment:
Using `util.inspect` and `JSON.strinigify` are not equal. Even the most simplest case doesn’t work.

```
util.inspect({ b: 'b'})
'{ b: \'b\' }'
JSON.parse(util.inspect({ b: 'b'}))
SyntaxError: Unexpected token b in JSON at position 2
```

It also says in the docs: `The util.inspect() method returns a string representation of object that is intended for debugging.` I don’t think we want to rely on that method.

Circular references would need to be handled differently, and it might not be the best for this method to handle them, because there are different ways you can handle it. So you just remove them?